Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+// MARK: - DeploymentTarget
+
+public enum DeploymentTarget: Codable {
+    case iOS(String, [DeploymentDevice])
+    case macOS(String)
+    // TODO: 🙈 Add `watchOS` and `tvOS` support
```

Comment:
Soon :)

Reply:
Actually, what platforms should we support in this PR? 🤔

Reply:
Should I leave this TODO or is it possible to support `watchOS` and `tvOS` right now?